docs/dev/libvirt/README.md

```diff
@@ -153,7 +153,7 @@ sudo firewall-cmd --zone=libvirt --add-service=libvirt
 
 NOTE: When the firewall rules are no longer needed, `sudo firewall-cmd --reload`
 will remove the changes made as they were not permanently added. For persistence,
-add `--permanent` to the `firewall-cmd` commands and run them a second time.
+add `--permanent` argument to the `firewall-cmd` commands and run them a second time.
```

Comment:
I'm fine with the previous line.  If you want to elaborate, I'd prefer "the `--permanent` option".

Reply:
ping @praveenkumar